cmd/cluster-display/main.go

```diff
@@ -34,7 +34,8 @@ func gatherOptions() (options, error) {
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 	fs.StringVar(&o.logLevel, "log-level", "info", "Level at which to log output.")
 	fs.IntVar(&o.port, "port", 8090, "Port to run the server on")
-	fs.StringVar(&o.hiveKubeconfigPath, "hive-kubeconfig", "", "Path to the kubeconfig file to use for requests to Hive.")
+	o.kubernetesOptions.NOInClusterConfigDefault = true
```

Comment:
Need to fix the deployment since we changed the arg name.

Reply:
No breaking changes. Support both flags, then migrate.

Reply:
Updated.